ui/libproject/devstart.rb

```diff
@@ -89,6 +96,12 @@ def clean_git_hooks()
 })
 
 Common.register_command({
+  :invocation => "rebuild-image",
```

Comment:
That's the main reason I have endeavored to provide this "executable documentation" which is less-likely to become incorrect.